File: src/main/java/com/lambdaschool/zoos/ZoosApplication.java
package com.lambdaschool.zoos;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class ZoosApplication {

    public static void main(String[] args) {
        SpringApplication.run(ZoosApplication.class, args);
    }

}

File: src/main/java/com/jakeesveld/zoos/ZoosApplication.kt
package com.jakeesveld.zoos

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
open class ZoosApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(ZoosApplication::class.java, *args)
        }

    }
}

